.github/workflows/Helix-CI.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       with:
         java-version: 1.8
     - name: Build with Maven
-      run: mvn clean install -Dmaven.test.skip.exec=true
+      run: mvn clean install -Dmaven.test.skip.exec=true -DretryFailedDeploymentCount=5
     - name: Run All Tests
-      run: mvn -q -fae test
+      run: mvn -q -fae -Dsurefire.rerunFailingTestsCount=5 test
```

Comment:
Does this rerun for the test class or the test case?

Reply:
Just that test case